ddtrace/tracer/sqlcomment_test.go

```diff
@@ -26,6 +26,7 @@ func TestSQLCommentCarrier(t *testing.T) {
 		mode               DBMPropagationMode
 		injectSpan         bool
 		samplingPriority   int
+		spanTags           map[string]string
```

Comment:
This test is currently not passing.

this field should be removed, and instead you need to add the 2 newly added `peerHostname` and `peerDBName` here (previous tests can be just empty strings), and pass them to the `SQLCommentCarrier` in line 151